cmd/soroban-cli/src/commands/contract/alias/remove.rs

```diff
@@ -0,0 +1,72 @@
+use std::fmt::Debug;
+
+use clap::{command, Parser};
+
+use crate::commands::{config::network, global};
+use crate::config::locator;
+use crate::print::Print;
+
+#[derive(Parser, Debug, Clone)]
+// #[command(group(
+//     clap::ArgGroup::new("wasm_src")
+//         .required(true)
+//         .args(&["wasm", "wasm_hash"]),
+// ))]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub config_locator: locator::Args,
+
+    #[command(flatten)]
+    network: network::Args,
+
+    /// The contract alias that will be removed.
+    pub alias: String,
+}
+
+#[derive(thiserror::Error, Debug)]
+pub enum Error {
+    #[error(transparent)]
+    Locator(#[from] locator::Error),
+
+    #[error(transparent)]
+    Network(#[from] network::Error),
+
+    #[error("no contract found with alias `{alias}`")]
+    NoContract { alias: String },
+}
+
+impl Cmd {
+    #[allow(clippy::unused_async)]
+    pub async fn run(&self, global_args: &global::Args) -> Result<(), Error> {
+        let print = Print::new(global_args.quiet);
+        let alias = &self.alias;
+        let network = self.network.get(&self.config_locator)?;
+
+        print.infoln(format!(
+            "Network passphrase: {passphrase}",
+            passphrase = network.network_passphrase
+        ));
+
+        let contract = self
+            .config_locator
+            .get_contract_id(&self.alias, &network.network_passphrase)?;
+
+        if contract.is_none() {
+            return Err(Error::NoContract {
+                alias: alias.into(),
+            });
+        };
+
+        let contract = contract.expect("contract must be set");
```

Comment:
We've had bugs arise from the use of expect/unwrap vs safe unwrapping. Safe unwrapping is a small modification on this flow:
```suggestion
        let Some(contract) = contract else {
            return Err(Error::NoContract {
                alias: alias.into(),
            });
        };
```